e2e/test/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -189,6 +191,32 @@ describe("scenarios > admin > settings", () => {
     cy.get(".cellData").and("contain", "2025/2/11, 9:40 PM");
   });
 
+  it("should show where to display the unit of currency (metabase#table-metadata-missing-38021 and update the formatting", () => {
+    // Set the semantic type of total to currency
+    cy.request("PUT", `/api/field/${ORDERS.TOTAL}`, {
+      semantic_type: "type/Currency",
+    });
+
+    cy.visit(
+      `/admin/datamodel/database/${SAMPLE_DB_ID}/schema/${SAMPLE_DB_SCHEMA_ID}/table/${ORDERS_ID}/field/${ORDERS.TOTAL}/formatting`,
+    );
+
+    cy.findByTestId("admin-layout-content").within(() => {
+      // Assert that this option now exists
+      cy.findByText("Where to display the unit of currency");
+      cy.findByText("In every table cell").click();
+    });
+
+    // Open the orders table
+    openOrdersTable({ limit: 2 });
+
+    cy.get("#main-data-grid").within(() => {
+      // Items in the total column should have a leading dollar sign
+      cy.findByText("$39.72");
+      cy.findByText("$117.03");
```

Comment:
The row order will be stable right? If it's not this might flake. 
I suspect it's fine though.